guides/learning/booking_learning_time.md

```diff
@@ -25,19 +25,19 @@ We want to ensure teams own their learning time and that includes thinking about
 - Ask your team members if it's okay with them, including your delivery manager/lead
 - Ask your customers if it's okay with them (for non-billable roles, ensure internal projects won’t be adversely affected)
 - Ensure your responsibilities are covered whilst you're away
-- Ensure that the personal learning time is placed into CharlieHR
+- Ensure that the personal learning time is placed into HiBob
 
 Ask your line manager, the Ops Team, and the People Team, if you wish to book personal learning time outside of the guidelines
 
 ## How do I record this time?
 
 Just like booking a holiday
 
-Log into CharlieHR
+Log into HiBob
 
-Navigate to book time off, (once it’s been okayed  by everyone) and then choose Learning Time from the drop down menu.
+Click Request Time Off (on the home page) or My Time Off (on the Time page), (once it’s been okayed  by everyone) and then choose Learning Time from the drop down menu.
```

Comment:
The 2nd bit is nearly there on the journey, technically it's also still "Request time off" from the time tab once you've selected "my time off". Both of those routes then bring you to the same screen - can you add that additional step in your wording?